micmac_li3ds/import_autocal.py

```diff
@@ -2,9 +2,8 @@
 import logging
 
 from cliff.command import Command
-from argparse import Namespace
 
-from . import api as li3ds
+from . import api as Api
```

Comment:
By convention module names start with a lower-case character, while class names start with an upper-case character. So please use something else than `Api` here for the alias to the `api` namespace. You probably don't need an alias anyway. You could use `api` to refer to the module name, and `apiobj` for the `api.Api` instance.